modules/client/examples/05-encoders/04-withdraw.ts

```diff
@@ -0,0 +1,25 @@
+/* MARKDOWN
+### Withdrawal
+
+Withdraws assets from a given DAO and transfers them to another address.
+In order for a withdrawal to be successful, the address executing it must have `WITHDRAW` permissions.
+*/
+
+import { Client, IWithdrawParams } from "@aragon/sdk-client";
+import { context } from "../00-setup/00-getting-started";
+
+// Instantiates an aragonOSx SDK client.
+const client: Client = new Client(context);
+
+const withdrawParams: IWithdrawParams = {
+  recipientAddress: "0x1234567890123456789012345678901234567890",
+  amount: BigInt(10),
+  tokenAddress: "0x1234567890123456789012345678901234567890", // contract address of the token to be withdrawn. if left empty, ETH will be withdrawn.
+  reference: "reason for the withdrawal"
```

Comment:
This has changed, after the withdraw function no longer exists. Also, the parameters now specify what token type and there's no `reference` 🙏

Reply:
In this scenario though, I see that the `WithdrawParams` interface also doesn't have the `recepientAddress` attribute. How is it supposed to know where to withdraw the funds to? 
@brickpop @josemarinas